rgw/v2/tests/s3_swift/multisite_configs/test_datalog_trimming.yaml

```diff
@@ -0,0 +1,12 @@
+# upload type: non multipart
```

Comment:
can you add polarion id here as well

Reply:
Vivek, I think you have automated bi-log trimming here and not datalog trimming.

Reply:
Vivek, It would be good if you change the value of rgw_sync_log_trim_interval to some less value (say 5 mins) instead of the default one of 20 mins and test if that is working to trim the bi log entries.

I think 20 mins will be a very long wait.

Reply:
> can you add polarion id here as well

Done